lib/fluent/plugin/in_kubernetes_metrics_aggregator.rb

```diff
@@ -513,7 +523,7 @@ def process_node_response(response)
           router.emit generate_tag('node') << ('.cpu.utilization'), Fluent::EventTime.from_time(@scraped_node_at), 'node' => node_name, 'value' => node_cpu_utilization
           node_cpu_reservation = node_req_lim.instance_variable_get(:@cpu_request).to_f / node_cpu_allocatable
           router.emit generate_tag('node') << ('.cpu.reservation'), Fluent::EventTime.from_time(@scraped_node_at), 'node' => node_name, 'value' => node_cpu_reservation
-          node_memory_utilization = node_res_usage.instance_variable_get(:@memory_usage).to_f / 1_000_000 * node_memory_allocatable # converting from bytes to megabytes
+          node_memory_utilization = node_res_usage.instance_variable_get(:@memory_usage).to_f /  node_memory_allocatable # converting from bytes to megabytes
```

Comment:
Any reason you are moving the conversion to line #590. 
`pod_memory_usage = pod_json['memory'].fetch('usageBytes', 0) / 1024 ** 2 # Converting to Mi`